unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/WebSocketCommunication.cs

```diff
@@ -196,6 +196,7 @@ private void InitMessageTypeToBridgeName()
         messageTypeToBridgeName["UpdateTotalFriendRequests"] = "Main";
         messageTypeToBridgeName["UpdateTotalFriends"] = "Main";
         messageTypeToBridgeName["UpdateHomeScene"] = "Main";
+        messageTypeToBridgeName["InitializeMatrix"] = "Main";
```

Comment:
no need for this line if you move the `InitializeMatrixBridge` to the `Bridges` object